src/main/java/org/apache/commons/compress/archivers/ar/ArArchiveOutputStream.java

```diff
@@ -137,14 +137,15 @@ private long writeEntryHeader(final ArArchiveEntry pEntry) throws IOException {
         boolean mustAppendName = false;
 
         final String n = pEntry.getName();
-        if (LONGFILE_ERROR == longFileMode && n.length() > 16) {
+        final int nLength = n.length();
```

Comment:
I can't see much benefit from extracting `n.length()` to a final `nlength`. Maybe it's performance benefiting or something else?